frontend/lib/src/helper/truncate.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+/*
+  Reusable "truncate"
+  
+  The availableWidth parameter in the truncateString function is used to specify the
+  available screen width.
+
+  example: truncate(String, 20 ), // Specify the available screen width you want
+*/
+
+String truncate(String value, double availableWidth) {
```

Comment:
use named parameters for the function signature, so when its called , its like 
truncate(value: "xyz", maxWidth: 20)